flink-utils/src/main/java/org/hypertrace/core/flinkutils/avro/RegistryBasedAvroSerde.java

```diff
@@ -61,13 +61,14 @@ public boolean isEndOfStream(T nextElement) {
     return TypeInformation.of(this.clazz);
   }
 
+  /**
+   * @param element to serialize
+   * @return serialized bytes
+   * @throws org.apache.kafka.common.errors.SerializationException
+   */
   @Override
   public byte[] serialize(T element) {
-    try {
-      return serializer.serialize(topicName, element);
-    } catch (Exception e) {
-      return new byte[0];
-    }
+    return serializer.serialize(topicName, element);
```

Comment:
Can we catch, log and rethrow. That would at least log the exact Avro that caused the problem